contrib/debian/dogecoin-qt.install

```diff
@@ -1,6 +1,6 @@
-usr/local/bin/bitcoin-qt usr/bin
+usr/local/bin/dogecoin-qt usr/bin
 share/pixmaps/bitcoin32.xpm usr/share/pixmaps
 share/pixmaps/bitcoin16.xpm usr/share/pixmaps
 share/pixmaps/bitcoin128.png usr/share/pixmaps
-debian/bitcoin-qt.desktop usr/share/applications
-debian/bitcoin-qt.protocol usr/share/kde4/services/
+debian/dogecoin-qt.desktop usr/share/applications
+debian/dogecoin-qt.protocol usr/share/kde4/services/
```

Comment:
A little lost, new line where exactly? At the end of the file? Seems to have one to me :-/